core/math/bsp_tree.cpp

```diff
@@ -577,12 +577,11 @@ BSP_Tree::BSP_Tree(const PoolVector<Face3> &p_faces, real_t p_error_radius) {
 	error_radius = p_error_radius;
 }
 
-BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) {
-
-	nodes = p_nodes;
-	planes = p_planes;
-	aabb = p_aabb;
-	error_radius = p_error_radius;
+BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
```

Comment:
Thanks. I didn't have time to look into this until now and the information helped a lot.

I played around with those options and setting `BreakConstructorInitializersBeforeComma` to `true` gives the following output:

```cpp
BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
	: nodes(p_nodes)
	, planes(p_planes)
	, aabb(p_aabb)
	, error_radius(p_error_radius) {
}
```
which separates the tab and the space with a comma or a colon.

Tweaking the other constructor format options just puts each initializer on the same line. For some reason I can't reproduce the format in this pull request without `vim-clang-format` (which is probably a configuration problem from my end), but that format above is consistent with the Vim plugin and `clang-format`. Shall we use that format?